wellplated/models.py

```diff
@@ -136,15 +136,17 @@ class Container(ClusterableModel):
 
     wells: Manager['Well']
 
-    def __getattr__(self, label: str) -> 'Well':
-        # ClusterableModel or other packages need an AttributeError for the following:
-        # _cluster_related_objects
-        # _prefetched_objects_cache
-        # get_source_expressions
-        # resolve_expression
+    def __getattr__(self, label: str) -> 'Well | None':
+        # These attributes seem to be added after __init__ but raising an AttributeError for them
+        # breaks the the Django admin and Wagtail manage interfaces.
+        if label in ('code', 'format'):
+            return None
 
         label_match = LABEL_384.match(label)
         if not label_match or label_match.groupdict().keys() != {'row', 'column'}:
+            # ClusterableModel or other packages need an AttributeError for:
```

Comment:
> ClusterableModel or other packages

Is ClusterableModel a package or a type?

Also, can you document why the need an `AttributeError`?

Reply:
It's a Wagtailism

https://github.com/wagtail/django-modelcluster?tab=readme-ov-file#django-modelcluster

I don't really know why other than there's some specific exception squashing.